packages/astro/src/cli/add/index.ts

```diff
@@ -700,7 +700,7 @@ async function tryToInstallIntegrations({
 						...inheritedFlags,
 						...installCommand.dependencies,
 					],
-					{ cwd }
+					{ cwd, env: { NODE_ENV: 'development' } }
```

Comment:
Interesting! We actually set the `NODE_ENV` here so the config file is loaded correctly:

https://github.com/withastro/astro/blob/0d4922dffe7d96b907db89ec502f77efd572cc56/packages/astro/src/cli/index.ts#L127

On this line, could we set `NODE_ENV: undefined` instead to reset the value? Also would be great to add an additional comment explaining why we need it.

Reply:
the reset seems to work with `undefined`.
i also added a comment to explain why we are doing this 👍